File: PSET6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
aux = 0
for i in range(height):
    aux = aux+1
    print(" " * ((height-i)-1), end="")
    print("#" * aux, end="")
    print("  ", end="")
    print("#" * aux)

File: PSET1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printBricks(int brickNumber)
{
    int aux=0;

    for(int i=1;i<=brickNumber;i++)
    {
        aux=brickNumber;
        for(int j=1;j<=i;j++)
        {
            while(i<aux)    
            {
                printf(" ");
                aux=aux-1;
            }

            printf("#");
        }
        printf("  ");
        for(int k=1;k<=i;k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

int main(void)
{
    int nBricks;

    do
    {
        nBricks=get_int("Height: ");
    }
    while(nBricks>8 || nBricks<1);

    printBricks(nBricks);
}
